### ISSUE TITLE
make get-token fails with special characters in password

### ISSUE THREAD

**Issue by dkastl** [lets-fiware/FIWARE-Big-Bang#231]
**Checklist**
_Put an `x` in the boxes that apply_

-   [x] I have run the latest release.
-   [ ] I have inspeced the status of docker containers.
-   [ ] I have run the sanity check described in after-installation.md.
-   [x] I have inspected all error messages.
-   [ ] I have inspected the log files in the /var/log/fiware directory.
-   [x] I have run the `make collect` command.

**Describe the bug**

The `make get-token` command fails with the following error:

```sh
$ make get-token
./config/script/get_token.sh
parse error: Invalid numeric literal at line 1, column 59
make: *** [Makefile:30: get-token] Error 4
```

**Your system environment**

```
Date: Thu Jan 26 08:18:58 UTC 2023
Version: 0.20.0
Hash: 17816b305d1f5c9b1c87f06760780c86e233c27a44d0a53e8dc7f9a3b8ec98ba  lets-fiware.sh
git-hash: 
f6f3e2f81480d4e9b14c6a6681409a733ed49599
d938834fd90694140ebabdbb9e9591bae8263593
b16af4929dc3c0593b14996aa437c7ad8d266035
App list: KEYROCK CYGNUS WIRECLOUD NGSIPROXY IOTAGENT_JSON IOTAGENT_HTTP MOSQUITTO ELASTICSEARCH 
Install: completed
Docker containers: 
make[1]: Entering directory '/srv/fiware-big-bang'
sudo /usr/local/bin/docker-compose ps
               Name                             Command                   State                                                             Ports                                                      
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
fiware-big-bang_cygnus_1             /cygnus-entrypoint.sh            Up (unhealthy)   5050/tcp, 5080/tcp                                                                                              
fiware-big-bang_elasticsearch-db_1   /usr/local/bin/docker-entr ...   Restarting                                                                                                                       
fiware-big-bang_elasticsearch_1      /docker-entrypoint.sh elas ...   Up               9200/tcp, 9300/tcp                                                                                              
fiware-big-bang_iotagent-json_1      docker-entrypoint.sh node  ...   Up (healthy)     4041/tcp, 7896/tcp                                                                                              
fiware-big-bang_keyrock_1            docker-entrypoint.sh npm start   Up (healthy)     3000/tcp                                                                                                        
fiware-big-bang_memcached_1          docker-entrypoint.sh memca ...   Up               11211/tcp                                                                                                       
fiware-big-bang_mintaka_1            java -cp @/app/jib-classpa ...   Up               8080/tcp, 8443/tcp                                                                                              
fiware-big-bang_mongo_1              docker-entrypoint.sh --noj ...   Up               27017/tcp                                                                                                       
fiware-big-bang_mosquitto_1          /docker-entrypoint.sh /usr ...   Up               1883/tcp                                                                                                        
fiware-big-bang_nginx_1              /docker-entrypoint.sh ngin ...   Up               0.0.0.0:443->443/tcp,:::443->443/tcp, 0.0.0.0:80->80/tcp,:::80->80/tcp, 0.0.0.0:8883->8883/tcp,:::8883->8883/tcp
fiware-big-bang_ngsiproxy_1          docker/entrypoint.sh             Up                                                                                                                               
fiware-big-bang_orion-ld_1           orionld -fg -multiservice  ...   Restarting                                                                                                                       
fiware-big-bang_postgres_1           docker-entrypoint.sh postgres    Up               5432/tcp                                                                                                        
fiware-big-bang_timescale-db_1       docker-entrypoint.sh postg ...   Up (healthy)     5432/tcp                                                                                                        
fiware-big-bang_tokenproxy_1         docker-entrypoint.sh             Up               1029/tcp                                                                                                        
fiware-big-bang_wilma_1              docker-entrypoint.sh npm start   Up (unhealthy)   1027/tcp                                                                                                        
fiware-big-bang_wirecloud_1          /docker-entrypoint.sh            Up (healthy)     8000/tcp                                                                                                        
make[1]: Leaving directory '/srv/fiware-big-bang'
Keyrock: 
{"keyrock":{"version":"8.1.0","release_date":"2021-07-22","uptime":"00:05:49.4","git_hash":"https://github.com/ging/fiware-idm/releases/tag/8.1.0","doc":"https://fiware-idm.readthedocs.io/en/8.1.0/","api":{"version":"v1","link":"https://keyrock.xxxxxxxxx/v1"}}}
```

**To Reproduce**

Run the token request with:

```
curl -sS "https://keyrock.xxxxxxxxxxx/token" --data "username=admin@xxxxxxxxx" --data "password=%o6Go7nf64Lb%8Bp" | jq -r .access_token
```

This returns:

```
parse error: Invalid numeric literal at line 1, column 59
```

**Expected behavior**

Return an access token.

**Server (please complete the following information):**
- OS and OS version: Ubuntu 22.04
- Docker version: 20.10.23, build 7155243
- Docker-compose version: 1.29.2, build 5becea4c

**Additional context**

It worked with a password consisting only of numbers and alphabet characters.

**Comment by fisuda** [comment]
As a workaround, could you try the following?

Replace

```
curl -sS "https://${KEYROCK}/token" --data "username=${IDM_ADMIN_EMAIL}" --data "password=${IDM_ADMIN_PASS}" | jq -r .access_token
```
with
```
curl -sS "https://${KEYROCK}/token" --data "{\"username\": \"${IDM_ADMIN_EMAIL}\", \"password\": \"${IDM_ADMIN_PASS}\"}" --header "Content-Type: application/json" | jq -r .access_token
```
at `line 41` in `./config/script/get_token.sh.`

**Comment by fisuda** [comment]
Thank you for the feedback.

The reason for this error is that the `%` in the password was parsed as a URL escape while sending the request with `application/x-www-form-urlencoded`. The fix has been merged at PR #232.

p.s. If you think FIWARE Big Bang is the recommended tool for engineers, I would appreciate it if you could provide a Github Star to this repository. 

Thanks.
